voluncheer/static/css/map.css

```diff
@@ -40,5 +44,163 @@ body {
 }
 
 .text-center {
-    text-align: center!important;
+    text-align: center !important;
+}
+
+.org {
+    background-color: #4285F4;
```

Comment:
Can we add a border around the markers so that it is easier to differentiate when they are close together?